### PR TITLE
TASK: allow nullable dates

### PR DESCRIPTION
**Pull request by akleineb**
In order to allow nullable dates this change is necessary.

### CODE REVIEW

File: Classes/Indexer/Object/Transform/DateTransformer.php [hunk 1]
```diff
@@ -36,6 +36,14 @@ public function getTargetMappingType()
      */
     public function transformByAnnotation($source, TransformAnnotation $annotation)
     {
+        if ($source === null) {
+            return null;
+        }
+
+        if (is_string($source)) {
```
Comment (dfeyer) [inline Classes/Indexer/Object/Transform/DateTransformer.php]: Is this really related to "allow nullable date" ?

Comment (johannessteu) [reply Classes/Indexer/Object/Transform/DateTransformer.php]: Nope, the pr is based on an older version probably. This is an updated pr @dfeyer 
https://github.com/Flowpack/Flowpack.ElasticSearch/pull/79

**Comment by daniellienert** [comment]
Fixed with https://github.com/Flowpack/Flowpack.ElasticSearch/pull/79
